.changesets/maint_avery_refactor_router_events.md

```diff
@@ -0,0 +1,5 @@
+### chore: split out router events into its own module ([PR #3235](https://github.com/apollographql/router/pull/3235))
+
+Breaks down `./apollo-router/src/router.rs` into its own module `./apollo-router/src/mod.rs` with a sub-module `./apollo-router/src/events/mod.rs` that contains all of the streams that we combine to start a router (entitlement, schema, reload, configuration, shutdown, more streams to be added). This change makes adding new events/modifying existing events a bit easier since it's not in one huge giant file to rule them all.
```

Comment:
```suggestion
Breaks down `./apollo-router/src/router.rs` into its own module `./apollo-router/src/router/mod.rs` with a sub-module `./apollo-router/src/router/event/mod.rs` that contains all of the streams that we combine to start a router (entitlement, schema, reload, configuration, shutdown, more streams to be added). This change makes adding new events/modifying existing events a bit easier since it's not in one huge giant file to rule them all.
```